.gitignore

```diff
@@ -23,3 +23,7 @@ p2p_secret
 # Python
 __pycache__/
 *.py[cod]
+
+# bash scripts
+contracts/addresses.json
+contracts/env/prod
```

Comment:
Is this file missing in this PR or it does not exist yet?

Reply:
it's here so that no-one commits it by accident